artifact_utils.py

```
@@ -1,7 +1,12 @@
 import os
 
+
+def _artifact_dir():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_artifact_dir`

(ANN202)